src/components/bcros/dialog/CardedModal.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <UModal
+    :attach="attach || ''"
+    :model-value="display"
+    :data-cy="'bcros-dialog' + (name ? `-${name}` : '')"
+  >
+    <UCard
+      v-if="options"
+      :ui="{
+        header: {
+          background: 'bg-bcGovColor-darkBlue'
+        }
```

Comment:
The header needs a rounded top corners.  This is what we have for the freezeUnfreeze modal: `:ui="{ header: { background: 'bg-bcGovColor-darkBlue', base: 'font-2xl font-bold text-white rounded-t-lg' } }`.   Actually, I think we should use this CardedModal in the freezeUnfreeze component, but we can worry about this later.